Composer/ScriptHandler.php

```diff
@@ -426,19 +426,12 @@ protected static function getPhpArguments()
     protected static function getConsoleDir(Event $event, $actionName)
     {
         $options = static::getOptions($event);
+        $binDirExists = isset($options['symfony-bin-dir']) && static::hasDirectory($event, 'symfony-bin-dir', $options['symfony-bin-dir'], $actionName);
 
-        if (static::useNewDirectoryStructure($options)) {
-            if (!static::hasDirectory($event, 'symfony-bin-dir', $options['symfony-bin-dir'], $actionName)) {
-                return;
-            }
-
+        if ($binDirExists && file_exists($binDirExists.DIRECTORY_SEPARATOR.'console')) {
```

Comment:
This concatenation looks wrong, `$binDirExists` is a boolean not a path, am I missing something?